PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -1021,7 +1021,9 @@ bool SeqPair::ChangeSelectedBlock(design& caseNL) {
   }
   int newsel=rand() % caseNL.Blocks.at(anode).size();
   selected.at(anode)=newsel;
-  if(caseNL.GetBlockCounterpart(anode)!=-1) { selected.at( caseNL.GetBlockCounterpart(anode) )=newsel;}
+  //if(caseNL.GetBlockCounterpart(anode)!=-1) { 
+    //selected.at( caseNL.GetBlockCounterpart(anode) )=newsel;
+  //}
```

Comment:
These lines assign sympair the same selected index while blocks in the same pair may have different numbers of primitives. As these old code does not fit the new settings (different primitives can be in the same sympair), they are removed.